vllm/transformers_utils/configs/speculators_eagle.py

```diff
@@ -0,0 +1,352 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+
+import json
+import os
+from pathlib import Path
+from typing import Dict, Any, List, Optional, Union
+
+from transformers import PretrainedConfig
+
+from vllm.transformers_utils.configs.eagle import EAGLEConfig
+
+# Constants for speculators format
+SUPPORTED_SPECULATORS_TYPES = frozenset({"eagle", "eagle3"})
+DEFAULT_HIDDEN_SIZE = 4096
+DEFAULT_NUM_LOOKAHEAD_TOKENS = 5
+
+
+class SpeculatorsEagleConfig(EAGLEConfig):
```

Comment:
Why did we change this from SpeculatorsEagle config?

Reply:
To avoid confusion with SpeculativeConfig! we can think of a better name